### PR TITLE
Added last pause action reasons

### PR DESCRIPTION
**Pull request by zagovorichev**
Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/TCA-600
 
On the deliveryExecution request added last pause reasons info
 
#### How to test
 
Check `deliveryExecutions` request on the proctoring page, and find that there is a lastPauseReason data on the delivery executions which were paused by proctor

Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-proctoring/pull/821

### CODE REVIEW

File: model/execution/DeliveryExecutionList.php [hunk 1]
```diff
@@ -125,9 +129,43 @@ private function createExecution($cachedData, $extraFields)
...
     }
 
+    private function getLastPauseReason(string $deliveryExecutionId): ?array
+    {
+        $reason = null;
+        $lastPause = $this->getDeliveryLogService()->search([
+            DeliveryLog::DELIVERY_EXECUTION_ID => $deliveryExecutionId,
+            DeliveryLog::EVENT_ID => DeliveryLogEvent::EVENT_ID_TEST_PAUSE,
+        ], [
+            'order' => 'created_at',
```
Comment (SashaTAO) [inline model/execution/DeliveryExecutionList.php:149]: I think we have auto increment ID in this table. Check indexes please if it wouldn't be better to order by ID.

Comment (zagovorichev) [reply model/execution/DeliveryExecutionList.php:149]: Not agree that usage of the identifiers is better than usage of dates, when I want to order by date. I've checked and we have an index on the created_at field

**Comment by zagovorichev** [comment]
@SashaTAO @augustas Updated

File: test/unit/model/execution/DeliveryExecutionListTest.php [hunk 2]
```diff
@@ -251,6 +258,29 @@ public function testAdjustDeliveryExecutionsProgressStringWithCustomOption()
         $this->assertSame('finished', $result[0]['state']['progress']);
     }
 
+    public function testLatPauseReason(): void
```
Comment (SashaTAO) [inline test/unit/model/execution/DeliveryExecutionListTest.php:261]: Typo - `testLastPauseReason`

Comment (zagovorichev) [reply test/unit/model/execution/DeliveryExecutionListTest.php]: Fixed

**Pull request merged**